website/docs/r/cloud_aws_integrations.html.markdown

```diff
@@ -307,6 +309,67 @@ resource "newrelic_cloud_aws_integrations" "bar" {
+* `cloudtrail` - (Optional) AWS CloudTrail. See [Integration blocks](#integration-blocks) below for details.
+* `doc_db` - (Optional) AWS DocumentDB. See [Integration blocks](#integration-blocks) below for details.
+* `dynamodb` - (Optional) Amazon DynamoDB. See [Integration blocks](#integration-blocks) below for details.
+* `ec2` - (Optional) Amazon EC2. See [Integration blocks](#integration-blocks) below for details.
+* `ecs` - (Optional) Amazon ECS. See [Integration blocks](#integration-blocks) below for details.
+* `efs` - (Optional) Amazon EFS. See [Integration blocks](#integration-blocks) below for details.
+* `elasticache` - (Optional) AWS ElastiCache. See [Integration blocks](#integration-blocks) below for details.
+* `elasticbeanstalk` - (Optional) AWS Elastic Beanstalk. See [Integration blocks](#integration-blocks) below for details.
+* `elasticsearch` - (Optional) AWS ElasticSearch. See [Integration blocks](#integration-blocks) below for details.
+* `elb` - (Optional) AWS ELB (Classic). See [Integration blocks](#integration-blocks) below for details.
+* `emr` - (Optional) AWS EMR. See [Integration blocks](#integration-blocks) below for details.
+* `health` - (Optional) AWS Health. See [Integration blocks](#integration-blocks) below for details.
+* `iam` - (Optional) AWS IAM. See [Integration blocks](#integration-blocks) below for details.
+* `iot` - (Optional) AWS IoT. See [Integration blocks](#integration-blocks) below for details.
+* `kinesis_firehose` - (Optional) Amazon Kinesis Data Firehose. See [Integration blocks](#integration-blocks) below for details.
+* `lambda` - (Optional) AWS Lambda. See [Integration blocks](#integration-blocks) below for details.
+* `rds` - (Optional) Amazon RDS. See [Integration blocks](#integration-blocks) below for details.
+* `redshift` - (Optional) Amazon Redshift. See [Integration blocks](#integration-blocks) below for details.
+* `route53` - (Optional) Amazon Route 53. See [Integration blocks](#integration-blocks) below for details.
+* `s3` - (Optional) Amazon S3. See [Integration blocks](#integration-blocks) below for details.
+* `ses` - (Optional) Amazon SES. See [Integration blocks](#integration-blocks) below for details.
+* `sns` - (Optional) AWS SNS. See [Integration blocks](#integration-blocks) below for details.
+* `sqs` - (Optional) AWS SQS. See [Integration blocks](#integration-blocks) below for details.
+* `x_ray` - (Optional) AWS X-Ray. See [Integration blocks](#integration-blocks) below for details.
+x
+
+The following arguments are supported with minimum metrics polling interval of 900 seconds
+
+* `ebs` - (Optional) Amazon EBS. See [Integration blocks](#integration-blocks) below for details.
+* `kinesis` - (Optional) AWS Kinesis. See [Integration blocks](#integration-blocks) below for details.
+
+The following arguments are supported with minimum metrics polling interval of 3600 seconds
```

Comment:
```suggestion
The following arguments/integration blocks are intended to be used with a minimum `metrics_polling_interval` of 3600 seconds.
```

Reply:
Thanks for the feedback @pranav-new-relic , updated the document